app/src/main/java/org/mozilla/fenix/settings/address/view/AddressEditorView.kt

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.address.view
+
+import mozilla.components.concept.storage.UpdatableAddressFields
+import mozilla.components.support.ktx.android.view.hideKeyboard
+import org.mozilla.fenix.databinding.FragmentAddressEditorBinding
+import org.mozilla.fenix.settings.address.interactor.AddressEditorInteractor
+
+/**
+ * Shows an address editor for adding or updating an address.
+ */
+class AddressEditorView(
+    private val binding: FragmentAddressEditorBinding,
+    private val interactor: AddressEditorInteractor
+) {
+
+    /**
+     * Binds the view.
+     */
+    fun bind() {
```

Comment:
I'm assuming there's a separate story for _editing_ an address and for displaying the current addresses?

Reply:
Ya.